packages/config/config/devices/0x0063/59341_zwa3017.json

```diff
@@ -0,0 +1,17 @@
+{
+	"manufacturer": "Enbrighten",
+	"manufacturerId": "0x0063",
+	"label": "59341 / ZWA3017",
+	"description": "In-Wall Toggle Dimmer, QFSW, 700S",
+	"devices": [
+		{
+			"productType": "0x4944",
+			"productId": "0x3431"
+		}
+	],
```

Comment:
This file has the same IDs as 59335_zwa3017.json. We need to merge them into one.

Reply:
@jascoproducts

Reply:
> This file has the same IDs as 59335_zwa3017.json. We need to merge them into one.

Yes, let's merge.

On 59341_zwa3017.json:
- Change the label to "59341 / 59335 / ZWA3017"
- Change the filename to "59341_59335_zwa3017"

Description will remain the same.

Go ahead and remove 59335_zwa3017.json.